scikeras/wrappers.py

```diff
@@ -1636,7 +1679,7 @@ def scorer(y_true, y_pred, **kwargs) -> float:
 
     def _validate_data(
         self, X=None, y=None, reset: bool = False, y_numeric: bool = False
-    ) -> Tuple[np.ndarray, Union[np.ndarray, None]]:
+    ) -> Tuple[np.ndarray, Optional[np.ndarray]]:
```

Comment:
I don't think this should be changed from `Union[..., None]` -> `Optional[...]`. Yes, they are technically equivalent, but semantically `Optional` does not make sense if there is no default argument.